packages/arb-token-bridge-ui/src/components/TransferPanel/TokenImportDialog.tsx

```diff
@@ -43,17 +47,31 @@ type TokenListSearchResult =
       status: ImportStatus
     }
 
+type TokenImportDialogStore = {
+  isOpen: boolean
+  openDialog: () => void
+  closeDialog: () => void
+}
+
+export const useTokenImportDialogStore = create<TokenImportDialogStore>(
+  set => ({
+    isOpen: false,
+    openDialog: () => set({ isOpen: true }),
+    closeDialog: () => set({ isOpen: false })
+  })
+)
+
 export type TokenImportDialogProps = UseDialogProps & { tokenAddress: string }
```

Comment:
nit: 
```
export type TokenImportDialogProps = Omit<UseDialogProps, 'isOpen'> & { tokenAddress: string }
```

and then only use `TokenImportDialogProps` so the naming is more accurate